libs/aws/langchain_aws/chat_models/bedrock_converse.py

```diff
@@ -495,13 +498,15 @@ def _generate(
     ) -> ChatResult:
         """Top Level call"""
         bedrock_messages, system = _messages_to_bedrock(messages)
+        # logger.info(f"The input message sent by user: {bedrock_messages}")
```

Comment:
Please do not add commented code. If you want for it to be called only for fewer times, use `.debug()`